src/main/java/net/atos/zac/flowable/ZaakVariabelenService.java

```diff
@@ -48,11 +50,14 @@ public class ZaakVariabelenService {
     @Inject
     private CmmnRuntimeService cmmnRuntimeService;
 
+    @Inject
+    private CmmnHistoryService cmmnHistoryService;
+
     @Inject
     private RuntimeService bpmnRuntimeService;
 
     @Inject
-    private CmmnHistoryService cmmnHistoryService;
+    private HistoryService bpmnHistoryService;
```

Comment:
Constructor injection should be used where possible !

Reply:
This class did not make use of Constructor injection the moment I added the bpmnHistoryService. I'll leave this for now until there is a need to use the Constructor injection.